### PR TITLE
HW-382: Fixed UI snagging and tabs

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
Description:
This PR styles Individual case setting page keeping, global component of shoreditch in mind.

**Before:**
hw-382-before

**After:**
hw-382-after

**Technical Details:**
- Tabs were styled on two places (common, administrator) which we merged to have only one common style.
- Checkbox/radio placeholders are defined to allow individual checkbox/radio (i.e. without label) to have style applied

**Notes:**
- Checkbox/Radio is not styled globally keeping project time budget in mind but, it's coded the way it can be easily done in future and will extensive testing for that when doing.
- Table style is changed for padding globally

- [x] Backstop Tests passed

### CODE REVIEW

File: scss/civicrm/_mixins.scss [hunk 1]
```diff
@@ -36,4 +36,69 @@
 
 @mixin translate ($x, $y) {
   @include transform(translate($x, $y));
+}
+
+/*
+* The styled-checkbox() CSS function designs individual checkbox
+*/
+
```
Comment (AkA84) [inline scss/civicrm/_mixins.scss]: remove this empty line

File: scss/civicrm/_mixins.scss [hunk 2]
```diff
@@ -36,4 +36,69 @@
...
+    left: 2px;
+    top: 6px;
+    margin-top: -5px;
+  }
+}
+
+/*
+* The styled-checkbox() CSS function designs individual radio
+*/
+
```
Comment (AkA84) [inline scss/civicrm/_mixins.scss]: remove this empty line

File: scss/civicrm/_mixins.scss [hunk 3]
```diff
@@ -36,4 +36,69 @@
...
+/*
+* The styled-checkbox() CSS function designs individual checkbox
+*/
+
+@mixin styled-checkbox() {
+  height: 16px;
+  position: relative;
+
+  &::before {
+    content: '';
```
Comment (AkA84) [inline scss/civicrm/_mixins.scss]: please list the properties in alphabetical order (this is a general rule, if you haven't followed it so far, please start now)

File: scss/civicrm/_mixins.scss [hunk 4]
```diff
@@ -36,4 +36,69 @@
...
+    display: inline-block;
+    position: absolute;
+    top: 4px;
+    left: 4px;
+    width: 8px;
+    height: 8px;
+    background-color: $crm-copy;
+    border-radius: 50%;
+  }
 }
```
Comment (AkA84) [inline scss/civicrm/_mixins.scss]: please set up your IDE to leave a new line in each file

File: scss/civicrm/administer/civi-case/_case-types.scss [hunk 5]
```diff
@@ -7,11 +7,41 @@
       padding: 20px;
     }
 
+    input[type=checkbox] {
```
Comment (AkA84) [inline scss/civicrm/administer/civi-case/_case-types.scss]: i'm not sure what happened to the previous conversation, but I still don't know why we have CiviCase styles in shoreditch

File: scss/civicrm/common/_tabs.scss [hunk 6]
```diff
@@ -1,40 +1,82 @@
...
+      font-size: $font-size-base;
+      font-family: $font-family-base;
+      padding: 7px 15px;
+      margin-bottom: -1px;
+
+      &.crm-tab-button {
+        top: 0;
+      }
+
+      span.crm-i {
```
Comment (AkA84) [inline scss/civicrm/common/_tabs.scss]: do you need to specify `span`? If possible never specify the element in the selector

File: scss/civicrm/common/_tabs.scss [hunk 7]
```diff
@@ -1,40 +1,82 @@
...
+        color: $crm-h3-color;
+        font-family: $font-family-base;
+        font-size: $font-size-base;
+        font-weight: $crm-font-weight-h3;
+        padding-left: 5px;
+        margin: 4px 0px;
```
Comment (AkA84) [inline scss/civicrm/common/_tabs.scss]: `0` doesn't need units

File: scss/civicrm/common/_tabs.scss (hunk 6 above)
Comment (mukeshcompucorp) [reply scss/civicrm/common/_tabs.scss]: Yes, this one is necessary evil

**Pull request merged**